change/@fluentui-react-menu-a6169786-dc45-4b6d-b92f-ab803b7d6c0b.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31409&commitId=e0ac9987e3eeb5f3d3471668eabbcc246cd69444&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|5| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|
|Avatar Converged.image-bad-url+icon.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Menu Converged - selection</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|383| Changed|
|Menu Converged - selection.switch - Dark Mode.selected.chromium.png|438| Changed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|87| Changed|
|Menu Converged - selection.switch - High Contrast.selected.chromium.png|216| Changed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>